wls-wahlvorbereitung-service/src/test/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/rest/unterbrechungsuhrzeit/UnterbrechungsUhrzeitControllerIntegrationTest.java

```diff
@@ -0,0 +1,212 @@
+            val request1 = buildPostRequest(wahlbezirkID, writeDto1);
+
+            mockMvc.perform(request1).andExpect(status().isCreated());
+            SecurityUtils.runAs(Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT);
+            val unterbrechungsUhrzeitFromRepo1 = unterbrechungsUhrzeitRepository.findById(wahlbezirkID).get();
+            val expectedUnterbrechungsUhrzeit1 = unterbrechungsUhrzeitModelMapper.toEntity(unterbrechungsUhrzeitDTOMapper.toModel(wahlbezirkID, writeDto1));
+
+            Assertions.assertThat(unterbrechungsUhrzeitFromRepo1).isEqualTo(expectedUnterbrechungsUhrzeit1);
+
+            val writeDto2 = new UnterbrechungsUhrzeitWriteDTO(LocalDateTime.now().truncatedTo(ChronoUnit.MILLIS));
+            val request2 = buildPostRequest(wahlbezirkID, writeDto2);
+
+            mockMvc.perform(request2).andExpect(status().isCreated());
+
+            SecurityUtils.runAs(Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT);
+            val unterbrechungsUhrzeitFromRepo2 = unterbrechungsUhrzeitRepository.findById(wahlbezirkID).get();
+            val expectedUnterbrechungsUhrzeit2 = unterbrechungsUhrzeitModelMapper.toEntity(unterbrechungsUhrzeitDTOMapper.toModel(wahlbezirkID, writeDto2));
+
+            Assertions.assertThat(unterbrechungsUhrzeitFromRepo2).isEqualTo(expectedUnterbrechungsUhrzeit2);
+        }
+
+        @Test
+        @WithMockUser(
+                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT,
+                        Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT }
+        )
+        void gotWlsExceptionWhenParameterNotComplete() throws Exception {
+            val wahlbezirkID = "wahlbezirkID";
+            val writeDto = new UnterbrechungsUhrzeitWriteDTO(null);
+            val request = buildPostRequest(wahlbezirkID, writeDto);
+
+            val response = mockMvc.perform(request).andExpect(status().isBadRequest()).andReturn();
+            val exceptionBodyFromResponse = objectMapper.readValue(response.getResponse().getContentAsString(StandardCharsets.UTF_8), WlsExceptionDTO.class);
+
+            SecurityUtils.runAs(Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT);
+            Assertions.assertThat(unterbrechungsUhrzeitRepository.findById(wahlbezirkID)).isEmpty();
+
+            val expectedExceptionDTO = new WlsExceptionDTO(WlsExceptionCategory.F, ExceptionConstants.PARAMS_UNVOLLSTAENDIG.code(), "WLS-WAHLVORBEREITUNG",
+                    ExceptionConstants.PARAMS_UNVOLLSTAENDIG.message());
+            Assertions.assertThat(exceptionBodyFromResponse).usingRecursiveComparison().isEqualTo(expectedExceptionDTO);
+        }
+
+        @Test
+        @WithMockUser(
+                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT,
+                        Authorities.REPOSITORY_READ_UNTERBRECHUNGSUHRZEIT }
+        )
```

Comment:
Das Read-Recht wird erst nach dem mockMVC.perform benötigt und dort wirt es auch explizit gesetzt
```suggestion
        @WithMockUser(
                authorities = { Authorities.SERVICE_UNTERBRECHUNGSUHRZEIT, Authorities.REPOSITORY_WRITE_UNTERBRECHUNGSUHRZEIT }
        )
```